notebooks/_03-Vision.md

```diff
@@ -18,8 +18,7 @@ jupyter:
 
 ## At the highest level, how to solve this project’s problem---what will be done in the project? 
 
-
-In this project, we want to demonstrate using simulations how to structurally approach demand modeling and modeling in general while paying attention to the causal structure of how people make their choices, and the data generation process in general. 
+We will demonstrate using simulations how to structurally approach demand modeling and modeling in general while paying attention to the causal structure of how people make their choices, and the data generation process in general. 
 We will build on Brathwaite’s work and review paper and demonstrate with steps how to incorporate and think about causal inference in the context of DCM. We will also demonstrate the value of such an approach by contrasting the outcome of this way of thinking to that of traditional demand modeling where little care is given to the causal structure. 
 
 Specifically, there will be two main deliverables in our work/presentation. The first one is a simple application of the steps outlined in Brathwaite et. al to a simulation example, where we will also show what would happen to our causal estimates under different assumptions of the causal graph. In this example, the problem would be purely a selection on observables problem, and there won’t be complications related to unobserved latent confounders -- the goal is to solely demonstrate the step-by-step approach with explicitly stating and showing the causal assumptions being made. 
```

Comment:
So if I'm understanding correctly, the first part is simply showcasing a causal graph for a realistic simulation example, in the context of selection on observables.

Will we deal with variable selection here or no? We don't need to deal with it, I'm just raising the question.

Variable selection is probably the only thing that computationally will differ in the selection on observables setting.

Reply:
For line 26, where it says "(Maybe) apply this approach to a real world dataset as well," why maybe? It seems that this is the most externally valued part of the project.

Reply:
Lastly, for part 2, is there also going to be a comparison to a traditional DCM approach for omitted variables? It wasn't specified.

Reply:
> Lastly, for part 2, is there also going to be a comparison to a traditional DCM approach for omitted variables? It wasn't specified.

@timothyb0912 I think it only makes sense to have the comparison. Just a quick question: do you think it isn't necessary? If so, why?

Reply:
Alright, resolving convo based on video call: We'll mention DCM approaches to omitted variables in the talk but not show a comparison in the presentation.